src/transforms/filter.js

```diff
@@ -129,6 +158,30 @@ function getFilterFunc(opts) {
             return function(v) { return v < value; };
         case '>':
             return function(v) { return v > value; };
```

Comment:
will try to get an example up, but yes it does work with datetime strings.  It does not seem to work though with `Date` and datetime strings.